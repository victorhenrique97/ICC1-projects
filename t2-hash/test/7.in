541
500
166
476
404
378
324
607
847
582
913
440
264
672
257
788
954
845
354
407
174
958
639
486
279
249
432
918
789
185
952
714
430
477
669
356
931
178
770
421
791
891
547
245
154
437
823
522
991
905
564
773
694
615
165
229
567
896
362
990
580
270
874
632
196
778
764
346
376
677
839
381
833
114
419
560
948
291
350
382
810
827
584
849
802
396
512
348
187
420
691
779
529
637
794
930
322
108
893
101
716
233
206
819
863
268
877
604
784
182
611
684
400
685
572
643
570
697
532
406
638
436
330
719
443
335
237
433
777
917
749
821
310
762
815
152
129
922
191
364
846
374
968
703
956
369
292
972
488
934
545
255
131
975
151
629
518
793
490
851
115
109
447
723
320
831
156
242
744
646
581
589
225
725
141
825
681
280
219
732
454
299
843
941
755
606
955
375
357
351
914
390
359
125
741
121
113
366
410
228
325
385
230
423
885
667
944
392
925
534
665
888
513
286
300
747
769
664
837
622
135
124
926
605
205
828
247
150
894
494
671
790
946
633
860
483
383
977
238
224
195
149
288
573
602
439
222
841
650
640
119
599
294
289
981
284
401
675
961
331
910
689
493
414
130
104
623
536
593
645
942
641
110
901
140
193
561
649
556
702
143
915
464
585
204
393
258
1000
576
428
722
692
935
647
969
882
176
504
526
835
337
163
183
367
117
920
340
543
315
696
251
679
852
192
519
349
391
481
499
776
844
887
334
267
673
368
394
122
916
345
906
707
820
616
617
103
128
753
656
102
469
936
621
670
866
252
203
370
949
107
553
562
651
980
134
184
781
727
516
338
507
328
517
148
455
145
482
487
700
661
842
855
610
986
200
422
730
498
767
157
243
285
295
254
256
480
363
215
603
210
813
998
787
248
899
395
775
737
479
510
180
736
999
693
817
904
188
951
771
202
995
642
415
932
814
690
976
281
277
765
403
133
881
644
170
278
574
478
409
636
132
539
505
168
186
388
858
253
290
508
832
997
528
785
261
554
668
921
709
654
327
655
175
466
705
173
597
227
805
489
760
260
731
612
427
426
434
491
596
624
577
558
592
726
250
939
360
830
262
754
786
780
979
907
386
158
269
578
412
992
971
276
308
715
861
571
266
974
938
808
634
146
