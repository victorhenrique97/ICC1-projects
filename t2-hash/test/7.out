0:
1:
2: 543
3:
4: 545
5:
6: 547
7:
8:
9:
10:
11:
12: 553
13: 554
14:
15: 556
16:
17: 558
18:
19: 560
20: 561
21: 562
22:
23: 564
24:
25:
26: 567
27:
28:
29: 570
30: 571
31: 572
32: 573
33: 574
34:
35: 576
36: 577
37: 578
38:
39: 580
40: 581
41: 582
42:
43: 584
44: 585
45:
46:
47:
48: 589
49:
50:
51: 592
52: 593
53:
54:
55: 596
56: 597
57:
58: 599
59:
60:
61: 602
62: 603
63: 604
64: 605
65: 606
66: 607
67:
68:
69: 610
70: 611
71: 612
72:
73:
74: 615
75: 616
76: 617
77:
78:
79:
80: 621
81: 622
82: 623
83: 624
84:
85:
86:
87:
88: 629
89:
90:
91: 632
92: 633
93: 634
94:
95: 636
96: 637
97: 638
98: 639
99: 640
100: 641
101: 101 642
102: 643 102
103: 103 644
104: 104 645
105: 646
106: 647
107: 107
108: 108 649
109: 109 650
110: 110 651
111:
112:
113: 113 654
114: 114 655
115: 115 656
116:
117: 117
118:
119: 119
120: 661
121: 121
122: 122
123: 664
124: 665 124
125: 125
126: 667
127: 668
128: 669 128
129: 129 670
130: 671 130
131: 672 131
132: 673 132
133: 133
134: 675 134
135: 135
136: 677
137:
138: 679
139:
140: 681 140
141: 141
142:
143: 684 143
144: 685
145: 145
146: 146
147:
148: 689 148
149: 149 690
150: 691 150
151: 151 692
152: 152 693
153: 694
154: 154
155: 696
156: 697 156
157: 157
158: 158
159: 700
160:
161: 702
162: 703
163: 163
164: 705
165: 165
166: 166 707
167:
168: 168 709
169:
170: 170
171:
172:
173: 714 173
174: 174 715
175: 716 175
176: 176
177:
178: 178 719
179:
180: 180
181: 722
182: 182 723
183: 183
184: 725 184
185: 185 726
186: 727 186
187: 187
188: 188
189: 730
190: 731
191: 191 732
192: 192
193: 193
194:
195: 195 736
196: 196 737
197:
198:
199:
200: 741 200
201:
202: 202
203: 744 203
204: 204
205: 205
206: 206 747
207:
208: 749
209:
210: 210
211:
212: 753
213: 754
214: 755
215: 215
216:
217:
218:
219: 219 760
220:
221: 762
222: 222
223: 764
224: 224 765
225: 225
226: 767
227: 227
228: 228 769
229: 770 229
230: 230 771
231:
232: 773
233: 233
234: 775
235: 776
236: 777
237: 778 237
238: 779 238
239: 780
240: 781
241:
242: 242
243: 784 243
244: 785
245: 245 786
246: 787
247: 788 247
248: 789 248
249: 249 790
250: 791 250
251: 251
252: 793 252
253: 794 253
254: 254
255: 255
256: 256
257: 257
258: 258
259:
260: 260
261: 802 261
262: 262
263:
264: 264 805
265:
266: 266
267: 267 808
268: 268
269: 810 269
270: 270
271:
272: 813
273: 814
274: 815
275:
276: 817 276
277: 277
278: 819 278
279: 279 820
280: 821 280
281: 281
282: 823
283:
284: 825 284
285: 285
286: 827 286
287: 828
288: 288
289: 289 830
290: 831 290
291: 291 832
292: 833 292
293:
294: 294 835
295: 295
296: 837
297:
298: 839
299: 299
300: 300 841
301: 842
302: 843
303: 844
304: 845
305: 846
306: 847
307:
308: 849 308
309:
310: 310 851
311: 852
312:
313:
314: 855
315: 315
316:
317: 858
318:
319: 860
320: 320 861
321:
322: 322 863
323:
324: 324
325: 325 866
326:
327: 327
328: 328
329:
330: 330
331: 331
332:
333: 874
334: 334
335: 335
336: 877
337: 337
338: 338
339:
340: 340 881
341: 882
342:
343:
344: 885
345: 345
346: 346 887
347: 888
348: 348
349: 349
350: 891 350
351: 351
352: 893
353: 894
354: 354
355: 896
356: 356
357: 357
358: 899
359: 359
360: 901 360
361:
362: 362
363: 363 904
364: 905 364
365: 906
366: 366 907
367: 367
368: 368
369: 369 910
370: 370
371:
372: 913
373: 914
374: 374 915
375: 375 916
376: 376 917
377: 918
378: 378
379: 920
380: 921
381: 381 922
382: 382
383: 383
384: 925
385: 385 926
386: 386
387:
388: 388
389: 930
390: 931 390
391: 391 932
392: 392
393: 934 393
394: 935 394
395: 936 395
396: 396
397: 938
398: 939
399:
400: 400 941
401: 401 942
402:
403: 944 403
404: 404
405: 946
406: 406
407: 407 948
408: 949
409: 409
410: 410 951
411: 952
412: 412
413: 954
414: 955 414
415: 956 415
416:
417: 958
418:
419: 419
420: 420 961
421: 421
422: 422
423: 423
424:
425:
426: 426
427: 968 427
428: 428 969
429:
430: 430 971
431: 972
432: 432
433: 433 974
434: 975 434
435: 976
436: 436 977
437: 437
438: 979
439: 439 980
440: 440 981
441:
442:
443: 443
444:
445: 986
446:
447: 447
448:
449: 990
450: 991
451: 992
452:
453:
454: 454 995
455: 455
456: 997
457: 998
458: 999
459: 1000
460:
461:
462:
463:
464: 464
465:
466: 466
467:
468:
469: 469
470:
471:
472:
473:
474:
475:
476: 476
477: 477
478: 478
479: 479
480: 480
481: 481
482: 482
483: 483
484:
485:
486: 486
487: 487
488: 488
489: 489
490: 490
491: 491
492:
493: 493
494: 494
495:
496:
497:
498: 498
499: 499
500:
501:
502:
503:
504: 504
505: 505
506:
507: 507
508: 508
509:
510: 510
511:
512: 512
513: 513
514:
515:
516: 516
517: 517
518: 518
519: 519
520:
521:
522: 522
523:
524:
525:
526: 526
527:
528: 528
529: 529
530:
531:
532: 532
533:
534: 534
535:
536: 536
537:
538:
539: 539
540:
